docs/2_2_common_schema.adoc

```diff
@@ -20,7 +20,7 @@ With the enterprise edition of XMLSpy it is possible to automatically generate C
 C# and Java code that reads an XML file of fmiModelDescription.xsd.
 An efficient open source XML parser is SAX (http://sax.sourceforge.net/, http://en.wikipedia.org/wiki/Simple_API_for_XML).
 All data from the XML file is only defined via "attributes" and not via "elements".
-Therefore, only an "attribute" handler needs to be defined for a SAX parser.].
+Therefore, only an "attribute" handler needs to be defined for a SAX parser.] except for comments. If comments within the above mentioned schema files are inconsistent with the standard text, then the standard text is to be considered normative.
```

Comment:
There should be no more comments in the XSDs. If so, they should be removed.

Reply:
I found that there are no comments in the XSDs, so I will remove this. Will update soon